packages/financial-templates-lib/test/clients/FinancialContractEventClient.js

```diff
@@ -75,7 +66,7 @@ const versionedIt = function (supportedVersions, shouldBeItOnly = false) {
 };
 
 contract("FinancialContractEventClient.js", function (accounts) {
-  for (let tokenConfig of configs) {
+  for (let tokenConfig of POSSIBLE_TEST_DECIMAL_COMBOS) {
```

Comment:
We can remove references to older versions in the client now! [example](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/clients/FinancialContractEventClient.js#L325)